.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Rule change request

```

Comment:
I'm a bit confused about this one. The file name says feature request, which usually means new rule or new core feature, but the front matter says rule change request.